tests/test_geospatialscenario.py

```diff
@@ -184,16 +193,18 @@ def test_downselect_CONUS(monkeypatch):
         data=[
             [-99, -99, -1, "+100", "United States", "Alaska", "filler", 2],
             [-99, -99, -1, "+100", "United States", "Hawaii", "filler", 2],
+            [-99, -99, -1, "+100", "United States", "Alaska", "filler", 2],
+            [-99, -99, -1, "+100", "United States", "Hawaii", "filler", 2],
         ],
         columns=[
-            'latitude',
-            'longitude',
-            'altitude',
-            'tz',
-            'country',
-            'state',
-            'county',
-            'wind_height'
+            "latitude",
+            "longitude",
+            "altitude",
+            "tz",
+            "country",
+            "state",
+            "county",
+            "wind_height",
```

Comment:
Remove the trailing comma on the last item?
```suggestion
            "wind_height"
```

Reply:
My black formatter keeps adding them...